libs/ledger-live-common/src/bridge/impl.ts

```diff
@@ -53,28 +53,9 @@ export const getAccountBridge = (
   if (type === "mock") {
     const mockBridge = mockBridges[currency.family];
     if (mockBridge) return mockBridge.accountBridge;
-    throw new CurrencyNotSupported(
-      "no mock implementation available for currency " + currency.id,
-      {
-        currencyName: currency.name,
-      }
-    );
```

Comment:
yes, we should have fallback on the real implementation in the current new implementation otherwise it raises errors in some unit tests.

some coins don't have mock implementation. 
since now, when we call "applyReconciliation" or "toAccountRaw" or "fromAccountRaw", we always retrieve "accountBridge" firstly. ( compared to the old implementation, we call such functions directly without getting "accountBridge"). So we should make sure that we can access the "accountBridge" even if there is no mock implementation of "accountBridge" for such coins.